Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.cpp

```diff
@@ -51,24 +51,18 @@ void InitKeyboardMouse(IDirectInput8* const idi8, HWND _hwnd)
       SUCCEEDED(mo_device->SetDataFormat(&c_dfDIMouse2)) &&
       SUCCEEDED(mo_device->SetCooperativeLevel(nullptr, DISCL_BACKGROUND | DISCL_NONEXCLUSIVE)))
   {
-    g_controller_interface.AddDevice(std::make_shared<KeyboardMouse>(kb_device, mo_device));
+    g_controller_interface.AddDevice(
+        std::make_shared<KeyboardMouse>(kb_device.Get(), mo_device.Get()));
```

Comment:
What happens to this pointer once `kb_device` and `mo_device` go out-of-scope in this method? Wouldn't they be free'd, since you pass a raw pointer (where no ref-counting happens)?